{"name": "test_001", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <scripts.test_case.TestCase object at 0x000001797A38F790>\n\n    def test_001(self):\n>       assert 0\nE       assert 0\n\nscripts\\test_case.py:3: AssertionError"}, "start": 1748149014981, "stop": 1748149014981, "uuid": "42c61c90-6ae5-4385-8fc0-fe716716df14", "historyId": "b52620e2fa88e5ec1efe61251c01560f", "testCaseId": "b52620e2fa88e5ec1efe61251c01560f", "fullName": "scripts.test_case.TestCase#test_001", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "LAPTOP-47HNDVHS"}, {"name": "thread", "value": "25548-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_case"}]}